menu/drivers/xmb.c

```diff
@@ -4435,6 +4438,46 @@ static int xmb_list_push(void *data, void *userdata,
             {
                entry.enum_idx      = MENU_ENUM_LABEL_LOAD_CONTENT_LIST;
                menu_displaylist_ctl(DISPLAYLIST_SETTING_ENUM, &entry);
+
+               const struct retro_subsystem_info* subsystem = NULL;
+               subsystem = system->subsystem.data;
+               if (subsystem)
+               {
+                  for (i = 0; i < system->subsystem.size; i++, subsystem++)
+                  {
+                     char s[PATH_MAX_LENGTH];
+                     if (content_get_subsystem() == i)
+                     {
+                        if (content_get_subsystem_rom_id() < subsystem->num_roms)
+                        {
+                           snprintf(s, sizeof(s), "Load %s %s", subsystem->desc, i == content_get_subsystem() ? "\u2605" : " ");
+                           menu_entries_append_enum(info->list,
+                                 s,
+                                 msg_hash_to_str(MENU_ENUM_LABEL_SUBSYSTEM_ADD),
+                                 MENU_ENUM_LABEL_SUBSYSTEM_ADD,
+                                 MENU_SETTINGS_SUBSYSTEM_ADD + i, 0, 0);
+                        }
+                        else
+                        {
+                           snprintf(s, sizeof(s), "Start %s %s", subsystem->desc, i == content_get_subsystem() ? "\u2605" : " ");
+                           menu_entries_append_enum(info->list,
+                                 s,
+                                 msg_hash_to_str(MENU_ENUM_LABEL_SUBSYSTEM_LOAD),
+                                 MENU_ENUM_LABEL_SUBSYSTEM_LOAD,
+                                 MENU_SETTINGS_SUBSYSTEM_LOAD, 0, 0);
+                        }
+                     }
+                     else
+                     {
+                        snprintf(s, sizeof(s), "Load %s %s", subsystem->desc, i == content_get_subsystem() ? "\u2605" : " ");
+                        menu_entries_append_enum(info->list,
```

Comment:
this adds the Load Subsystem entry to the main menu for the any not subsystem that is not currently active

Actually I just realized this code will do nothing here:
**i == content_get_subsystem() ? "\u2605" : " "**